1.0.0 changes

cs-abhishek-shukla opened the pull request:
- Added configuration instruction for ServiceNow
- Other grammar and typo checks:

cs-rohit-khune commented on docs/setup.md:
```diff
@@ -21,4 +21,7 @@ The **FortiRecon EASM** solution pack depends on the following solution packs th
+    - To configure and use the Fortinet FortiRecon EASM connector, refer to [Configuring Fortinet FortiRecon EASM](https://docs.fortinet.com/fortisoar/connectors/fortinet-fortirecon-easm)
+
+- ServiceNow connector to log a ticket for fixing certificate issues detected by FortiRecon EASM
+    - To configure and use the Fortinet FortiRecon EASM connector, refer to [Configuring Fortinet FortiRecon EASM](https://docs.fortinet.com/fortisoar/connectors/servicenow-v2-0-1)
```
Please replace "Fortinet FortiRecon EASM" with "ServiceNow" on line 27

cs-abhishek-shukla committed:
1.0.0 changes
- Added configuration instruction for ServiceNow
- Other grammar and typo checks

cs-rohit-khune reviewed (approved):
Approved
